modules/502-delivery/openapi/openapi-case-tests.yaml

```
@@ -0,0 +1,7 @@
+positive:
+  values:
+    - auth:
+        webui:
```

```suggestion
        webui:
```
Looks like a copy paste from the upmeter module and IDK how it works.

---

Removed, test cases fulfilled